File: view/console/ClientView.php
<?php
require_once "../../model/Client.php";
class ClientView{
    private ClientService $clientService;
    public function __construct(){
        $this->clientService = new ClientService();
    }

    public static function  saisitClient():Client{
        $client=new Client();
        $client->setId(readline("Vueillez entrer l'id: "));
        $client->setNom(readline("Vueillez entrer le nom: "));
        $client->setPrenom(readline("Vueillez entrer le prenom: "));
        $client->setTelephone(readline("Vueillez entrer le telephone: "));
        $client->setObservation(readline("Vueillez entrer votre observation: "));
        $client->setAdresse(readline("Vueillez entrer votre adresse: "));
        return $client;
    }
    public static function afficheClient(array $clients):void{
        foreach($clients as $client){
            echo "ID: {$client->getId()}, Nom: {$client->getNom()}, Prénom: {$client->getPrenom()}, Téléphone: {$client->getTelephone()}, Observation: {$client->getObservation()}\n";
        }
    }

}

File: atelier_couture/src/vues/ClientView.java
package vues;

import data.entites.Client;
import services.ClientService;

public class ClientView extends AbstractView{

    private ClientService clientService;

    public void creerClient(){
        Client client = new Client();
        System.out.print("nom ? ");
        String nom = scanner.nextLine();
        client.setNom(nom);
        System.out.print("prenom ? ");
        String prenom = scanner.nextLine();
        client.setPrenom(prenom);
        System.out.println("Adresse ?");
        String adresse = scanner.nextLine();
        client.setAdresse(adresse);
        System.out.println("Telephone ?");
        String telephone = scanner.nextLine();
        client.setTelephone(telephone);
        clientService.create(client);
    }

    public void listerClient(){
        for(Client client:clientService.getAll()){
            System.out.println(client);
        }
    }

}
